.pullapprove.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 requirements:
-  signed_off_by:
+  signed_off_by: Udo Hartmann <udo1toni@gmail.com>
```

Comment:
Please leave this file alone

Reply:
https://pullapprove.com/openhab/openhab2-addons/pull-request/2710/

Reply:
I suspect I did not understand how to sign off the correct way.
Please bring light to the darkness...